src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -246,8 +246,6 @@ private void saveFile(File fileToSave) {
     FileUtils.setUserDirectory(fileToSave.getParent());
     // create a rawConfig object from GUI content and serialize it as json
     JsonParser.writeToFile(fileToSave, createRawContestConfig());
-    // Reload to keep GUI fields updated in case invalid values are replaced during save process
-    loadFile(fileToSave, true);
```

Comment:
This shouldn't cause double log messages anymore like you mentioned in #43, because I added the silent mode (hence the extra `true` boolean). Please re-add it, because there are some subtle things it can correct for, just to make sure the GUI is fully synced with what's saved to the HDD.

Reply:
Tried this and silent mode does indeed work.